src/commands/plugins/install.ts

```diff
@@ -135,39 +137,39 @@ e.g. If you have a core plugin that has a 'hello' command, installing a user-ins
   async run(): Promise<void> {
     const {argv, flags} = await this.parse(PluginsInstall)
     this.flags = flags
-    if (flags.verbose && !flags.silent) this.plugins.verbose = true
-    if (flags.silent && !flags.verbose) this.plugins.silent = true
+
+    const plugins = new Plugins({
+      config: this.config,
+      verbose: this.flags.verbose,
+    })
     const aliases = this.config.pjson.oclif.aliases || {}
     for (let name of argv as string[]) {
       if (aliases[name] === null) this.error(`${name} is blocked`)
       name = aliases[name] || name
-      const p = await this.parsePlugin(name)
+      const p = await this.parsePlugin(plugins, name)
       let plugin
       await this.config.runHook('plugins:preinstall', {
         plugin: p,
       })
       try {
         if (p.type === 'npm') {
-          ux.action.start(`Installing plugin ${chalk.cyan(this.plugins.friendlyName(p.name) + '@' + p.tag)}`)
-          plugin = await this.plugins.install(p.name, {
+          ux.action.start(
+            `${this.config.name}: Installing plugin ${chalk.cyan(plugins.friendlyName(p.name) + '@' + p.tag)}`,
+          )
+          plugin = await plugins.install(p.name, {
             force: flags.force,
             tag: p.tag,
           })
         } else {
-          ux.action.start(`Installing plugin ${chalk.cyan(p.url)}`)
-          plugin = await this.plugins.install(p.url, {force: flags.force})
+          ux.action.start(`${this.config.name}: Installing plugin ${chalk.cyan(p.url)}`)
+          plugin = await plugins.install(p.url, {force: flags.force})
         }
       } catch (error) {
         ux.action.stop(chalk.bold.red('failed'))
-        YarnMessagesCache.getInstance().flush(plugin)
         throw error
       }
 
       ux.action.stop(`installed v${plugin.version}`)
-
-      YarnMessagesCache.getInstance().flush(plugin)
-
-      this.log(chalk.green(`\nSuccessfully installed ${plugin.name} v${plugin.version}`))
```

Comment:
Did you mean to remove this log message?

Reply:
Yes. The spinner now shows the success (or failure) message

```
@salesforce/cli: Installing plugin @oclif/plugin-version@latest... installed v2.0.14
```